### ISSUE TITLE
Feature Request: There is no file selector

### ISSUE THREAD

**Issue by a8underscore** [browsh-org/browsh#418]
Lots of websites, for example file sharing websites, will use a file selector to allow the user to select the file, this selector is lacking in browsh

**Comment by tombh** [comment]
Is this related to #235?

**Comment by a8underscore** [comment]
no, i just do not have the ability to select files to be uploaded
